apps/backend/app/agent/manager.py

```
@@ -38,24 +60,29 @@ async def run(self, prompt: str, **kwargs: Any) -> Dict[str, Any]:
+    ) -> EmbeddingProvider:
+        match self._model_provider:
+            case 'openai':
+                from .providers.openai import OpenAIEmbeddingProvider
+                api_key = kwargs.get("openai_api_key", settings.LLM_API_KEY)
+                return OpenAIEmbeddingProvider(api_key=api_key)
+            case 'ollama':
+                from .providers.ollama import OllamaEmbeddingProvider
+                model = kwargs.get("embedding_model", self._model)
+                return OllamaEmbeddingProvider(embedding_model=model)
+            case _:
+                from .providers.llama_index import LlamaIndexEmbeddingProvider
```

Ruff (0.12.2)

`.providers.llama_index.LlamaIndexEmbeddingProvider` imported but unused

Remove unused import: `.providers.llama_index.LlamaIndexEmbeddingProvider`

(F401)